coordinator/src/services/handlers/spine-handler.ts

```diff
@@ -124,15 +124,15 @@ export class LiveRequestHandler implements RequestHandler {
 
   private getSpineUrlForPrescription() {
     if (this.spineEndpoint.includes("ref")) {
-      return `${SPINE_URL_SCHEME}://${this.spineEndpoint.replace(/demographics/g, "prescriptions")}/${this.spinePath}`
+      return `${SPINE_URL_SCHEME}://${this.spineEndpoint.replace(/msg/g, "prescriptions")}/${this.spinePath}`
```

Comment:
This is a little bit hooky still. Is there no way we can just inject the correct URL to start with?

Reply:
Yep, we should have our own secret in EPS namespace, just patching to unblock ref for now